src/main/java/stsjorbsmod/actions/RepressedMemoryAction.java

```diff
@@ -0,0 +1,35 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.vfx.combat.FlashAtkImgEffect;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.patches.ExertedField;
+import stsjorbsmod.util.CombatUtils;
+
+public class RepressedMemoryAction extends AbstractGameAction {
```

Comment:
I'm not sure this action is altogether necessary. Either just do everything in the card's `use` or in the card's action. I prefer the former.